lib/tepee.rb

```diff
@@ -10,7 +10,8 @@ class << self
 
     protected
 
-    def add(name, default = nil, env_var: "#{env_var_prefix}#{name}")
+    def add(name, default = nil, options = {})
+      env_var = options[:env_var] || "#{env_var_prefix}#{name}"
```

Comment:
Hey ! Thanks for that. However, we don't want the add method to accept any kind of input (a typo to `env_var` would go unnoticed).